torch/csrc/autograd/functions/errors.h

```diff
@@ -0,0 +1,9 @@
+#ifndef PYTORCH_AUTOGRAD_ERRORS_H
+#define PYTORCH_AUTOGRAD_ERRORS_H
+
+#define PT_ERR_BACKWARD_TWICE "Trying to backward through the " \
```

Comment:
i'm afraid of const char* for global strings, if two libraries refer to this file in the future, there will be symbol conflicts.